components/AvalancheCenterSelector.tsx

```diff
@@ -59,50 +59,50 @@ export const AvalancheCenterCard: React.FunctionComponent<AvalancheCenterCardPro
   }
   if (isError) {
     return (
-      <Box>
-        <Box bg="light.100">
+      <View>
```

Comment:
`Box` had to be one of the more useless NativeBase wrappers. It's just a `View`!